HoloJS/HoloJsHost/ScriptsLoader.cpp

```diff
@@ -97,7 +100,10 @@ ScriptsLoader::LoadScripts(
 			wstring scriptText = scriptPlatformString->Data();
 			if (!scriptText.empty())
 			{
-				m_loadedScripts.emplace_back(std::move(scriptText));
+				Script* script = new Script();
```

Comment:
This won't merge properly using auto merge - it does not compile. I added a case above this code for downloading scripts if an absolute URL is given; the new code needs to be given the same treatment. Could you please merge master